neural_compressor/torch/quantization/load_entry.py

```diff
@@ -25,43 +25,63 @@
     RTNConfig,
     TEQConfig,
 )
+from neural_compressor.torch.utils import LoadFormat
 
 config_name_mapping = {
     FP8_QUANT: FP8Config,
 }
 
 
-def load(output_dir="./saved_results", model=None):
-    """The main entry of load for all algorithms.
+def load(model=None, checkpoint_dir="./saved_results", format="default", *hf_model_args, **hf_model_kwargs):
```

Comment:
checkpoint_dir is always required and please set it as args
`def load(checkpoint_dir, model=None, format="default", *hf_model_args, **hf_model_kwargs)`
I don't expect to change all load behavior in this PR.

Reply:
Updated to new design, now is `load(model_name_or_path, original_model=None, format='default', device='cpu', *model_args, **kwargs)`